app/controllers/tags_controller.rb

```diff
@@ -18,5 +18,6 @@ def trending
 
   def update_subscription
     current_user.tags = Tag.process_tags(params[:tags])
+    render json: {}, status: 200
```

Comment:
I know it's not meant to be your task, but could you help write a jbuilder to return all the tags the user is now subscribed to?